tests/integration_test.rs

```diff
@@ -44,20 +43,17 @@ fn test_parse_simple() {
 fn test_parse_compilation_metrics() {
     let expected_files = [
         "0_0_1/dynamo_output_graph",
-        "0_0_1/dynamo_guards",
         "0_0_1/compilation_metrics",
         "1_0_1/dynamo_output_graph",
-        "1_0_1/dynamo_guards",
         "1_0_1/compilation_metrics",
         "2_0_0/dynamo_output_graph",
-        "2_0_0/dynamo_guards",
         "2_0_0/compilation_metrics",
         "index.html",
         "failures_and_restarts.html",
     ];
     // Read the test file
-    // simple.log was generated from the following:
-    // TORCH_TRACE=~/trace_logs/test python test/inductor/test_torchinductor.py  -k TORCH_TRACE=~/trace_logs/comp_metrics python test/dynamo/test_misc.py -k test_graph_break_compilation_metrics
+    // comp_metrics.log was generated from the following:
+    // TORCH_TRACE=~/trace_logs/comp_metrics python test/dynamo/test_misc.py -k test_graph_break_compilation_metrics
```

Comment:
FWIW, you don't have to regenerate the old logs, because it's important for us to be compatible with parsing old logs, so having old logs around is useful too.

Reply:
(I wouldn't change your code here, because this becomes successively less important as time goes on lol)